_targets.R

```diff
@@ -1095,11 +1095,17 @@ tar_target(pmtiles_rnet, {
   
   
   tar_target(save_outputs, {
+    length(pmtiles_rnet)
     message("Saving outputs for ", parameters$date_routing)
     
     saveRDS(od_commute_subset, "outputdata/od_commute_subset.Rds")
     saveRDS(zones_stats, "outputdata/zones_stats.Rds")
     saveRDS(school_stats, "outputdata/school_stats.Rds")
+    
+    file.copy("outputs/daysmetric.pmtiles","outputdata/daysmetric.pmtiles")
```

Comment:
why not:

```suggestion
    file.copy("outputs/daysmetric.pmtiles","outputdata/daysmetric.pmtiles")
```

?

Reply:
Or just put the files in `outputdata` directly to reduce n. lines of code and for consistency?